Src/AmrCore/AMReX_FillPatchUtil_I.H

```diff
@@ -848,19 +848,31 @@ InterpFromCoarseLevel (Array<MF*, AMREX_SPACEDIM> const& mf, IntVect const& ngho
         }
     }
 
-    const bool nested = ( AMREX_D_TERM(nghost[0] % ratio[0] == 0,
-                                    && nghost[1] % ratio[1] == 0,
-                                    && nghost[2] % ratio[2] == 0));
+    // If needed, adjust to fully overlap the coarse cells.
+    IntVect nghost_adj = nghost;
+    for (int d=0; d<AMREX_SPACEDIM; ++d) {
+        nghost_adj[d] += nghost[d] % ratio[d];
```

Comment:
I think this should be `if ((nghost[d] % ratio[d]) != 0) nghost_adj[d] += ratio[d] - nghost[d] % ratio[d];`

Reply:
Agreed.